zipkin/index.js

```diff
@@ -0,0 +1,137 @@
+
```

Comment:
I assume this will ultimately be it's own node module, and you will be the author? That would be cool :)